sui/faucet.js

```diff
@@ -31,6 +34,12 @@ if (require.main === module) {
     program
         .name('faucet')
         .addOption(new Option('--recipient <recipient>', 'recipient to request funds for'))
+        .addOption(
+            new Option(
+                '--minBalance <amount>',
+                'tokens will only be requested from the faucet if recipient balance is below the amount provided',
+            ).default('1'),
```

Comment:
does this cause a failure from the faucet?

Reply:
If used too many times it can get us rate limited. Furthermore, even if not rate limited, I am currently unable to used the faucet because it throws a 500 response. Hence why I opened this PR